cli/script/command-executor.ts

```diff
@@ -657,7 +739,69 @@ function printDeploymentHistory(command: cli.IDeploymentHistoryCommand, packageH
                     releaseTime,
                     packageObject.appVersion,
                     packageObject.isMandatory ? "Yes" : "No",
-                    packageObject.description ? wordwrap(30)(packageObject.description) : ""
+                    packageObject.description ? wordwrap(30)(packageObject.description) : "",
+                    getPackageMetricsString(packageObject)
+                ]);
+            });
+        });
+    }
+}
+
+function printDeploymentMetrics(command: cli.IDeploymentMetricsCommand, metrics: any): void {
+    var labelToMetricsMap: { [label: string] : DeploymentMetric } = {};
+    var totalActive: number = 0;
+    var versionRegex = /v(\d+)/;
+    
+    Object.keys(metrics).forEach((metricKey: string) => {
+        var parsedKey: string[] = metricKey.split(":");
```

Comment:
I feel like the format of the metric key is an implementation detail that should be hidden away on the server - I'm just conscious that this is not the most friendly format as we get other consumers of the SDK. Could we move this parsing onto the server side, and then just return objects that have `.active`, `.downloaded`, etc fields, which can in turn be documented in `rest-definitions.d.ts`? Note we'd likely have to do this anyway if we added this information to the deployment history route.

Sorry to be such a hassle :(